zou/app/blueprints/crud/person.py

```diff
@@ -88,6 +92,11 @@ def post_delete(self, instance_dict):
     def update_data(self, data, instance_id):
         if "password" in data:
             del data["password"]
+        if "departments" in data:
+            departments = [
+                Department.get(department_id) for department_id in data["departments"]
```

Comment:
It doesn't handle the case where the department_id matches nothing in the database.

Reply:
It now raises a 404 exception if one department_id is not present in the DB